modules/generic/src/test/scala/effects.scala

```diff
@@ -44,7 +45,7 @@ class GenericEffectMapping[F[_]: Sync](ref: SignallingRef[F, Int]) extends Gener
           // Compute a ValueCursor
           RootEffect.computeCursor("foo")((_, p, e) =>
             ref.update(_+1).as(
-              genericCursor(p, e, Struct(42, "hi"))
+              genericCursor(p, e, Struct(Wrapper(42), "hi"))
```

Comment:
I think a test for this would probably be better in the other test suite ... it doesn't have anything particularly to do with effects.